.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+
+    - name: Install dependencies
+      run: go mod tidy
+
+    - name: Build
+      run: go build ./...
+
+    - name: Run tests
+      run: go test ./... -v
+
+    - name: Run linter
+      uses: golangci/golangci-lint-action@v3
```

actionlint (1.7.4)

the runner of "golangci/golangci-lint-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)